easy-ui-react/src/DataGrid/Table.tsx

```diff
@@ -121,6 +122,13 @@ export function Table<C extends Column>(props: TableProps<C>) {
               ))}
             </RowGroup>
             <RowGroup as="tbody">
+              {collection.size === 0 && renderEmptyState && (
```

Comment:
**question**: when the collection is empty and the consumer doesnt pass `renderEmptyState`, what do we display?

Reply:
good question! I think we should have a default value for `renderEmptyState`